constructor/main.py

```diff
@@ -160,13 +161,6 @@ def main_build(
             continue
         if isinstance(info[key], str):
             info[key] = list(yield_lines(join(dir_path, info[key])))
-        if key == "virtual_specs":
-            for value in info[key]:
-                if not value.startswith("__"):
-                    raise ValueError(
-                        "'virtual_specs' can only include virtual package names like '__name', "
-                        f"but you supplied: {value}."
-                    )
```

Comment:
Dealt with this regex: https://github.com/conda/constructor/pull/943/files#diff-59d91bb2afaa59e20826ccdba174a446cbb8d4d81666606c7288a83d8811aeb4R164